backend/alembic/versions/6bdd563d49e1_create_table.py

```
@@ -0,0 +1,125 @@
+    sa.Column('email', sa.String(), nullable=False),
+    sa.Column('password', sa.String(), nullable=True),
+    sa.Column('google_id', sa.String(), nullable=True),
+    sa.Column('full_name', sa.String(), nullable=True),
+    sa.Column('address', sa.String(), nullable=True),
+    sa.Column('phone', sa.String(), nullable=True),
+    sa.Column('role', sa.String(), nullable=False),
+    sa.Column('created_at', sa.TIMESTAMP(), server_default=sa.text('now()'), nullable=True),
+    sa.Column('updated_at', sa.TIMESTAMP(), server_default=sa.text('now()'), nullable=True),
+    sa.Column('is_verified', sa.Boolean(), nullable=True),
+    sa.PrimaryKeyConstraint('user_id'),
+    sa.UniqueConstraint('google_id')
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)